frontend/conf/copr-frontend.sysusers.conf

```diff
@@ -0,0 +1,2 @@
+u! copr-fe - 'COPR frontend user' /usr/share/copr/coprs_frontend /bin/bash
```

Comment:
rpm 4.20.1 just landed to fedora41 (and will land to f42 in matter of hours) https://bodhi.fedoraproject.org/updates/?packages=rpm could you please then edit the conditions to include also f41?

also I tried this to install on rawhide yesterday and I ended installing it without copr-fe user or group... did you managed to get copr-fe user and group?

Reply:
also I tried this to install on rawhide yesterday and I ended installing it without copr-fe user or group... did you managed to get copr-fe user and group?

Reply:
yes. it worked for me.

Reply:
I'm afraid we need to `s/u!/u/` to make this work on Fedora <= 41.